package.json

```diff
@@ -33,13 +33,15 @@
     "node": ">=8.10"
   },
   "browserslist": "Node >= 8.10, > 0.5%, not OperaMini all, not dead",
+  "peerDependencies": {
+    "@apollo/client": "*"
```

Comment:
```suggestion
    "@apollo/client": "^3.0.0"
```

Reply:
It's debatable if the prerelease versions should be included in the peer dependency semver range, but at the very least we should specify a major version (not `*`).